fmriprep/workflows/fieldmap/unwarp.py

```diff
@@ -335,21 +336,21 @@ def init_pepolar_unwarp_wf(fmaps, bold_file, omp_nthreads, layout=None,
     qwarp.interface.num_threads = omp_nthreads
 
     workflow.connect([
-        (explicit_mask_epi, prepare_epi_opposite_wf, [('out_file', 'inputnode.ref_brain')]),
+        (inputnode, prepare_epi_opposite_wf, [('in_reference_brain', 'inputnode.ref_brain')]),
         (prepare_epi_opposite_wf, qwarp, [('outputnode.out_file', 'base_file')]),
     ])
 
     if usable_fieldmaps_matching_pe:
-        prepare_epi_matching_wf = init_prepare_epi_wf(ants_nthreads=ants_nthreads,
+        prepare_epi_matching_wf = init_prepare_epi_wf(ants_nthreads=omp_nthreads,
```

Comment:
This is a fix of leftover variable, that should have been changed in the previous refactor.